pkg/envoy/lds/rbac_test.go

```diff
@@ -45,27 +44,10 @@ func TestBuildRBACPolicyFromTrafficTarget(t *testing.T) {
 					},
 				},
 				Principals: []*xds_rbac.Principal{
-					{
-						Identifier: &xds_rbac.Principal_OrIds{
-							OrIds: &xds_rbac.Principal_Set{
-								Ids: []*xds_rbac.Principal{
-									rbac.GetAuthenticatedPrincipal("sa-2.ns-2.cluster.local"),
-								},
-							},
-						},
-					},
-					{
-						Identifier: &xds_rbac.Principal_OrIds{
-							OrIds: &xds_rbac.Principal_Set{
-								Ids: []*xds_rbac.Principal{
-									rbac.GetAuthenticatedPrincipal("sa-3.ns-3.cluster.local"),
-								},
-							},
-						},
-					},
```

Comment:
Are we no longer using OR by default? Could you refactor this in such a way that we continue to use OR based rules as before, and then remove the need for OR based rules in a separate change if necessary? Otherwise, it's a bit hard to know if this change is regressing the existing behavior as it involes test changes too.

Reply:
added comments on how OR is used by default